impeller/entity/entity_unittests.cc

```diff
@@ -1446,6 +1446,45 @@ TEST_P(EntityTest, DrawAtlasNoColor) {
   ASSERT_TRUE(OpenPlaygroundHere(std::move(e)));
 }
 
+TEST_P(EntityTest, F16WideGamutDrawAtlasPlus) {
```

Comment:
This isn't going to be wide gamut, the name of the test has to end with "WideGamut".  Can you add an assert that it is applied like the other ones please.

Reply:
Done